### PR TITLE
add Log Water Consumption Per Day onto Log Water Consumption Widget

### LINKED ISSUE #34: I want to be able to see the logs for a date
On the Log screen below the current consumption capture, I should see a log of the water I have consumed. It should default to the current date, but should allow me to select another date in the past. 

Consume the log api, display the consumption in time order

Depends on #33 

Wonderful_Mockup_WDW_Logs

### PR DESCRIPTION
**Pull request by CodeItQuick**
Closes #34 

Not sure the desired units of each log entry? Would it be cups of water or liters, or something else?

Questionable choice of data structures also? Maybe it should be an int tuple?

### CODE REVIEW

**Comment by CodeItQuick** [comment]
I would typically hide this behaviour behind a feature flag for what its worth

**Comment by CodeItQuick** [comment]
Maybe add another issue to make a feature flag + hook up to hardcoded API call?

**Comment by tbd-develop** [comment]
https://github.com/tbd-friends/water-drink-water/discussions/54

**Comment by CodeItQuick** [comment]
Feature flag is implemented.

File: Water Drink Water/src/clients/blazor.wa.tbd/Pages/Log.razor [hunk 1]
```diff
@@ -12,4 +12,54 @@
             <button class="btn btn-primary" onclick="@(async () => await LogConsumption())">Log</button>
         </div>
     </div>
+    @if (false) { // "feature flag"
+        <div class="row">
+            <div class="col-md-12">
+                <input id="log-date-picker" 
+                       type="date" 
+                       min="2020-01-01"
+                       max="2030-01-01"
```
Comment (tbd-develop) [inline Water Drink Water/src/clients/blazor.wa.tbd/Pages/Log.razor:21]: Where do these min / max values come from? Did we agree on these somewhere?

**Review by tbd-develop — APPROVED** [review]

File: Water Drink Water/src/clients/blazor.wa.tbd/Pages/Log.razor (hunk 1 above)
Comment (CodeItQuick) [reply Water Drink Water/src/clients/blazor.wa.tbd/Pages/Log.razor:21]: Think we discussed this on stream, but we did not agree on these values.

**Pull request merged**